### PR TITLE
By jotform hello google

### PR DESCRIPTION
**Pull request by berkay-karlik**
closes novitai/admin#28

won't display button if no link provided in post

### CODE REVIEW

File: components/PostArticle.vue [hunk 1]
```diff
@@ -84,6 +85,11 @@ export default {
       type: String,
       required: true
     }
+    // applyLink: {
```
Comment (sarpulas) [inline components/PostArticle.vue:88]: Not sure what this does, we should explain it or delete it

File: components/PostArticle.vue [hunk 2]
```diff
@@ -94,7 +100,14 @@ export default {
       })
     }
   }
+  // created() {
```
Comment (sarpulas) [inline components/PostArticle.vue:103]: Same as above, keep if there is a reason to keep it

**Review by sarpulas — CHANGES_REQUESTED** [review]
See the note on comments, other than that looks great

**Comment by sarpulas** [comment]
Note: https://github.com/novitai/novitai.github.io/pull/13/commits/fd6436cec2e9a1157b2da6a13f53ba42c21fd750 added to retain backwards compatibility. Too lazy to make new google forms for all job ads now and we don't want to shut them out

File: components/PostArticle.vue (hunk 1 above)
Comment (berkay-karlik) [reply components/PostArticle.vue]: remvoed

File: components/PostArticle.vue (hunk 2 above)
Comment (berkay-karlik) [reply components/PostArticle.vue]: removed, does nothing, was for debug.

**Review by sarpulas — APPROVED** [review]
lgtm

**Pull request merged**